src/WebJobs.Script.WebHost/Startup.cs

```diff
@@ -44,6 +44,7 @@ public void Configure(IApplicationBuilder app, IApplicationLifetime applicationL
                 app.UseDeveloperExceptionPage();
             }
 
+            app.UseWebSockets();
```

Comment:
This is an unrelated change, no?

Reply:
Unrelated to the title yes, but since both are for BlueRidge, I put them in the same PR.